pluggy.py

```diff
@@ -605,10 +623,10 @@ def varnames(func):
 
     assert "self" not in args  # best naming practises check?
     try:
-        cache["_varnames"] = args
+        cache["_varnames"] = args, defaults
     except TypeError:
```

Comment:
I know this code was here before already, but I'm curious in which situations can `cache["_varnames"]` raise a `TypeError`.